events/generic/ready.js

```diff
@@ -46,11 +51,11 @@ module.exports = class {
 			});
 			if(status[parseInt(i+1, 10)]) i++;
 			else i = 0;
-		}, 20000); // Every 20 seconds
+		}, 2e4); // Every 20 seconds
```

Comment:
I think it's a good idea in a program that requires a lot of performance management like in C/C++, and therefore requires relatively precise values. In our case I think it's not really necessary and it takes away clarity.